smoke/features/csi-driver-e2e.feature

```diff
@@ -43,19 +42,16 @@ Feature: SharedSecrets and SharedConfigMap
             | "openshift_csi_share_mount_requests_total" |
         Then the metrics value for configured resources should be "1"
     
-    @manual
+    @automated @slowest
     Scenario: shared resource data gets removed if permission removed : CSI-01-TC03
-        Given user has created <"resource type"> with <"shared resource">
+        Given user has created resource type with shared resource
             | resource type | shared resource    |
             | configmap     | "sharedconfigmaps" |
             |  secret       | "sharedsecrets"    |
-        And creates another project that will access the cluster scoped shared resource type that references the resource in the first project
-            | resource type | resource       |
-            | configmap     | "share-config" |
-            |  secret       | "my-secret"    |
-        And creates rbac for the service account to use the shared resource in the pod
+        When creates another project that will access the cluster scoped shared resource that references the resource created in the first project
+        And RBAC for the service account to use the sharedresource in its pod
         And creates a pod "my-csi-app-check" that used the sharedconfigmaps or sharedsecrets
-        Then pod "my-csi-app-check" should mount the data available in the shared resource
+        Then pod "my-csi-app-check" in the second project should mount the data test3 available in the "share-config"
         When user removes the share related rbac "use" permissions
         Then pod "my-csi-app-check" in the second project should remove the share related data from its volume
```

Comment:
```suggestion
        Then pod "my-csi-app-check" in the second project should have the share related data from its volume removed within the controller's relist interval.
```

Reply:
See https://github.com/openshift/csi-driver-shared-resource/blob/master/docs/config.md#configuration-file for the default and how the change that interval.